dm/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18.0-alpine3.15 as builder
```

Comment:
deployments/engine/docker/Dockerfile#L1 should be updated also.

Reply:
Done, `git grep 'golang:1.18'` now doesn't return anything anymore